### ISSUE TITLE
ResizeObserver is not defined (Gatsby)

### ISSUE THREAD

**Issue by undavide** [nerdyman/react-compare-slider#25]
I've used `react-compare-slider` for a Gatsby project, and to my surprise the build kept failing because of 

```
WebpackError: ReferenceError: ResizeObserver is not defined
  
  - react-compare-slider.esm.js:149 
    node_modules/react-compare-slider/dist/react-compare-slider.esm.js:149:26
  
  - react-compare-slider.esm.js:395 
    node_modules/react-compare-slider/dist/react-compare-slider.esm.js:395:1
```

The offending lines being:

```
const useResizeObserver = (ref, handler) => {
> 149 |   const observer = useRef(new ResizeObserver(([entry]) => handler(entry.contentRect)));
      |                          ^
  150 |   const observe = useCallback(() => {
  151 |     ref.current && observer.current.observe(ref.current);
  152 |   }, [ref]); // Bind/rebind observer when `handler` changes.


  WebpackError: ReferenceError: ResizeObserver is not defined
```

I've tried it all unsuccessfully polyfilling ResizeObserver in my Gatsby project, but the one thing that made it work was to 

`npm install resize-observer-polyfill --save` 

And then add 

`import ResizeObserver from 'resize-observer-polyfill';`

on line 2 of `node_modules/react-compare-slider/dist/react-compare-slider.esm.js`. That made it compile the website without errors.

I have no idea whether there were better ways to solve this directly in Gatsby without modifying your package (I haven't found them, but if anyone has ideas I'll be glad to try), I'm mentioning this in case somebody runs into the same issue.

Thanks!

**Comment by nerdyman** [comment]
Hi @undavide, thanks for opening the issue!

v2 of the library removed the built-in ResizeObvserver polyfill since it's now supported in all browsers but I totally forgot about SSR.

I've got a fix and will release a new version tonight.

**Comment by nerdyman** [comment]
@undavide This has been fixed in `2.0.2` :rocket:
